### PR TITLE
790 popup message delete super event

### LINKED ISSUE #790: Update the popup window that appears when deleting a super-event to state that all sub-events linked to the super event will also be deleted by this action.

### PR DESCRIPTION
**Pull request by minump**
## Description
Added deletion message in popup window. Checks isSuperEvent flag and display deletion message accordingly

**Fixes #790**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [x] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

**Comment by bingzhang** [comment]
can you make a branch from develop branch? and refer to the implementation of https://github.com/rokwire/events-manager/pull/793?

**Comment by minump** [comment]
> can you make a branch from develop branch? and refer to the implementation of #793?

reverted to develop branch

**Comment by bingzhang** [comment]
@minump please fix the conflicts, then we will merge it. thanks.

**Review by bingzhang — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -26,6 +27,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix the placeholder of subevent id. [#766](https://github.com/rokwire/events-manager/issues/766)
 - Fix indentation of removal sub events. [#785](https://github.com/rokwire/events-manager/issues/785)
 - Fix the error message on required field. [#783](https://github.com/rokwire/events-manager/issues/783)
+- Fixed error message when deleting subevent. [#799](https://github.com/rokwire/events-manager/issues/799)
```
Comment (bingzhang) [inline CHANGELOG.md:30]: can you remove those? I think they are not for this pr.

**Review by bingzhang — APPROVED** [review]

**Pull request merged**